addons/explosives/functions/fnc_defuseExplosive.sqf

```diff
@@ -17,7 +17,9 @@
 #include "script_component.hpp"
 EXPLODE_2_PVT(_this,_unit,_explosive);
 
-if (getNumber (ConfigFile >> "CfgAmmo" >> typeof _explosive >> "ACE_explodeOnDefuse") == 1) exitWith {
+_explode = random 1.0;
+if (getNumber (ConfigFile >> "CfgAmmo" >> typeOf _explosive >> "ACE_explodeOnDefuse") == 1 &&
+	_explode < GVAR(ExplodeOnDefuseChance)) exitWith {
```

Comment:
We could add ACE_ExplodeOnDefuseChance parameter for explosives that defaults to 0 when not defined!